src/notation/internal/notationconfiguration.cpp

```diff
@@ -266,7 +273,11 @@ async::Notification NotationConfiguration::backgroundChanged() const
 
 QColor NotationConfiguration::foregroundColor() const
 {
-    return settings()->value(FOREGROUND_COLOR).toQColor();
+    if (scoreInversionEnabled() && uiConfiguration()->isCurrentThemeHighContrast()) {
+        return QColor("#000000");
```

Comment:
Wouldn't that just be `Color::black`?

Reply:
This is QColor though, rather than mu::draw::Color.

Reply:
OK, then `QColor::black`

Reply:
There is no such thing as `QColor::black`
There is only a `QColor::black()`, which returns the black part of the color from the Cmyk coloring system.

Reply:
IMHO yes.